sortingAlgos.js

```diff
@@ -1,21 +1,23 @@
 function bubbleSort() {

```

Comment:
Thank you for the minute changes here, this was default formatting added by vscode.

Reply:
Welcome